wvd-templates/Create and provision WVD host pool/nestedtemplates/unmanagedDisks-customvhdvm.json

```diff
@@ -149,9 +160,7 @@
                 "hardwareProfile": {
                     "vmSize": "[parameters('rdshVmSize')]"
                 },
-                "availabilitySet": {
-                    "id": "[resourceId('Microsoft.Compute/availabilitySets/', concat(parameters('rdshPrefix'), 'availabilitySet'))]"
-                },
+                "availabilitySet": "[if(equals(parameters('enablePersistentDesktop'),bool('False')),variables('availabilitySetId'),json('null'))]",
```

Comment:
Same as unmanagedDisks-galleryvm.

Reply:
yep, that makes sense, I'll fix it.

Reply:
BTW, I found out why I did that way, there is no way to hide the id attribute without this extra logic and ARM does not accept an id=null value.